app/views/market_models/show.html.haml

```diff
@@ -1,17 +1,16 @@
 %h1
   = @market_model.name
-%table.interactions.table
+%table.interactions.table.table-bordered
   %tr
-    %th Stakeholder from
-    %th Stakeholder To
+    %th From stakeholder
+    %th To stakeholder
```

Comment:
@ChaelKruip @grdw Would it be more meaningful to rename "From stakeholder" and "To stakeholder" to "Payer" and "Payee" respectively? I'm struggling to think of a situation where "from" isn't paying "to", and these terms might be a bit easier to understand to novice users.